src/VSSTypeSanitizer.cpp

```diff
@@ -113,11 +126,16 @@ void VssDatabase::checkAndSanitizeType(jsoncons::json &meta, jsoncons::json &val
     else if (dt == "double") {
         checkNumTypes<double>(meta,val);
     }
+    else if (dt == "enum") { 
```

Comment:
When will this ever happen? Can you point to an example in VSS?

Reply:
good catch, I forget to remove it